daffodil-test/src/test/resources/org/apache/daffodil/section23/dfdl_functions/Functions.tdml

```diff
@@ -562,10 +562,10 @@
     <xs:element name="e_lowercase2" type="xs:string" dfdl:inputValueCalc="{ fn:lower-case('TeSt LoWeR') }" />
     <xs:element name="e_lowercase3" type="xs:string" dfdl:inputValueCalc="{ fn:lower-case('') }" />
 
-    <xs:element name="e_contains1" type="xs:boolean" dfdl:inputValueCalc="{ fn:contains('The quick brown fox.', 'quick') }" />
```

Comment:
This one surprised me. I have no objection to the change. I just wonder if I can be conscious enough about language to avoid things as common as this.

Reply:
Yeah, the tool definitely needs some refining to reduce false positives. It wouldn't surprise me if we continue to see them from time to time. But the fact that this codebase represents many years of development and there were only a handful of things that needed to be changed says this likely won't be something that will require many changes from our current development.

Note that in this particular case, the false positive was a a test that took off the `T` from `The `, which resulted in 'he'. Instead of changing just that one test, I decided to just change all tests using `the quick brown fox` so they were consistent. So this was pretty rare case, and could have been a much smaller change.